src/abstractions/base-handler.ts

```diff
@@ -44,9 +45,9 @@ export abstract class BaseHandler<TProps extends BaseHandlerProps, TState extend
     };
 
     static childContextTypes = {
-        DropdownOpen: React.PropTypes.bool,
-        DropdownOnHeaderClickCallback: React.PropTypes.func,
-        DropdownOnSectionClickCallback: React.PropTypes.func
+        DropdownOpen: PropTypes.bool,

```

Comment:
Not sure whether `PropTypes.bool.isRequired` should be here, but if these properties are required, we'll have to update the prop-types provided here.
@MartynasZilinskas?